addon/PageScraper.js

```diff
@@ -73,6 +73,7 @@ PageScraper.prototype = {
           };
           resolve([imageWithSize]);
         });
+        image.addEventListener("error", () => reject());
```

Comment:
I suppose this should be fine for now, and we'll get reports of the error now to better understand when it's failing.